explore/src/main/scala/explore/config/offsets/OffsetEditor.scala

```diff
@@ -98,10 +98,6 @@ object OffsetEditor {
                           generateCurrentGrid(params, props.pointCount, updateOffsets)
                             .unless_(props.hasOffsets)
                         }
-      _              <- useEffectWithDeps(params): params =>
```

Comment:
I didn't understand this section, so I'll just assume everything gets updated OK. 😄